docs/source-app/workflows/byoc/create_cluster.rst

```diff
@@ -0,0 +1,145 @@
+:orphan:
+
+.. _create_cluster:
+
+
+##################
+Create AWS cluster
+##################
+
+**Audience:** Users looking to create a cluster to run Lightning Apps on their own private cloud infrastructure.
+
+**Prereqs:** basic familiarity with cloud provider infrastructure management.
+
+.. note:: This feature is currently available for early access! To create your own clutser `contact us <mailto:product@lightning.ai?subject=I%20want%20to%20run%20on%20my%20private%20cloud!>`_.
+
+
+----
+
+*******************************************
+Step 1- Create roles and permissions on AWS
+*******************************************
+
+In this step you’ll be creating a role on your cloud provider that allows Lightning to manage resources on your behalf (for example, creating EC2 instances for your cluster).
+To do this you can use the AWS CLI or the AWS management console.
+
+You will only have to preform this step once, and the same role can be used to create multiple clusters.
+
+----
+
+.. raw:: html
+
+    <div class="display-card-container">
+        <div class="row">
+
+.. displayitem::
+   :header: Create role with AWS CLI
+   :description: Create role with AWS CLI
+   :col_css: col-md-4
+   :button_link: aws_cli.html
+   :height: 180
+   :tag: Basic
+
+.. displayitem::
+   :header: Create role with AWS console
+   :description: Create role with AWS console
+   :col_css: col-md-4
+   :button_link: aws_console.html
+   :height: 180
+   :tag: Basic
+
+.. raw:: html
+
+        </div>
+    </div>
+
+
+----
+
+
+****************************
+Step 2- Get ARN for new role
+****************************
+
+To start a cluster, Lightning will need the external-id you set in step 1, and the ARN role. Click on your new role to find them (note: you need the ARN listed at the top of the page, not the one in “trusted entitles”).
+
+
+Record the ARN and the external ID. You’ll need them for your next step.
+
+-----
+
+*******************************
+Create a Lightning BYOC cluster
```

Comment:
Is this step 3? it is the same header level as step 2

Reply:
Yes, this is Step 3. let me change that.